files/image_config/ntp/ntp.conf.j2

```diff
@@ -90,32 +138,24 @@ interface listen eth0
 {% endif %}
 interface listen 127.0.0.1
 
-# Access control configuration; see /usr/share/doc/ntp-doc/html/accopt.html for
-# details.  The web page <http://support.ntp.org/bin/view/Support/AccessRestrictions>
-# might also be helpful.
-#
-# Note that "restrict" applies to both servers and clients, so a configuration
-# that might be intended to block requests from certain clients could also end
-# up blocking replies from your own upstream servers.
+{# Access control options -#}
+{% set options = '' -%}
+
+{# Allow additional servers mobilization from the pool. Otherwise we don't need
+that -#}
+{% if is_pools == False -%}
```

Comment:
Note that because of Jinja's variable scoping rules, `is_pools` will always be `False`, and `nopeer` will always get added. See https://jinja.palletsprojects.com/en/2.10.x/templates/#assignments for more info on this.

As such, I think we would want `nopeer` to be present always in the default `restrict` configuration?

Reply:
No, it shoudn't be there always. I will check condition and change if needed